### DISCUSSION TITLE
Grouped Modules

### DISCUSSION THREAD

**Discussion by SirioAstarot** [joomla/joomla-cms#32424] [2. Cold Features]
**Add the option to group sets of modules in tree lists.**
This way of organizing the modules, simplifies the editions of our website, saving search time.

- Group Module 1
- - Module 1
- - Module 2
- Group Module 2
- - Module 1
- - - Module A
- - - Module B
- - - Module C
- - Module 2
- - - Module A
- - - Module B
- - - Module C

**Comment by 1337180** [comment]
nice idea, it wouldn't take much of coding and can be interpreted nicely, looks like a pure new option

**Reply by infograf768** [reply]
> it wouldn't take much of coding

If so easy, please provide a PR for 4.1 as a new feature.

**Comment by 1337180** [comment]
I'm not attacking you nor am I developer. I dig canalization, its so easy.

**Comment by chmst** [comment]
I also think that it is a nice idea and release leads of next version will welcome a PR for that.

**Comment by joomlaweby** [comment]
Nice idea. Cannot it be made with categories? Then we can control permissions per group (category) of modules.
